doc/src/content/components/testimonial/1.md

```diff
@@ -1,7 +1,7 @@
 ---
 path: "/testimonial/1"
 component: "testimonial"
-type: "molecule"
+type: "component"
```

Comment:
So any app specific molecule will be a component ? or 3rd level molecule will be a component ?

Reply:
Yes, you are right